src/fontra/client/core/vector.js

```diff
@@ -61,3 +65,7 @@ export function intersect(pt1, pt2, pt3, pt4) {
 export function distance(pt1, pt2) {
   return vectorLength(subVectors(pt2, pt1));
 }
+
+export function dotVector(point, point2) {
```

Comment:
Let's rename this `dotProduct()`, and the arguments `vectorA` and `vectorB` to keep the style consistend with the rest of the module.